package.json

```diff
@@ -19,20 +19,21 @@
     "riddle-exam": "file:packages/riddle-exam"
   },
   "devDependencies": {
-    "@eslint/eslintrc": "^3",
-    "@testing-library/cypress": "^10.0.3",
+    "@eslint/eslintrc": "3.2.0",
+    "@testing-library/cypress": "10.0.3",
     "@types/lodash": "^4.17.14",
     "@types/node": "^20",
     "@types/react": "^19",
     "@types/react-dom": "^19",
-    "@vitest/coverage-v8": "^2.1.8",
-    "@vitest/ui": "^2.1.8",
-    "cypress": "^13.17.0",
+    "@vitest/coverage-v8": "3.0.5",
+    "@vitest/ui": "3.0.5",
```

Comment:
upgraded a few dependencies to use the latest features. For instance, the `it.each()` function from Vitest